image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -139,6 +139,10 @@ collector:
     enabled: null # bool
   runtimeConfig:
     enabled: null # bool
+    networking:
+      externalIps:
+        enable: null # bool
+      perContainerRateLimit: null # int
```

Comment:
This is the maximum number of open networking connections reported by collector in one scrape interval. The default scrape interval is 30 seconds. @Stringy Can you come up with an alternative name that is clearer?

Reply:
Oh my, that's an elaborate unit 😅 
`maxConnsPerScrapeInterval` perhaps? Or `maxConnsReportedPerScrapeInterval` 🤔 

Or `maxConnsPerContainerInScrapeInterval`?

Perhaps drop the `Interval`?
Is the scrape interval user-configurable? Because alternatively, we could specify the per minute rate here and compute the value that goes into the config map? 🤔

Reply:
I am not sure that `maxConnsPerContainerInScrapeInterval` is clearer than `perContainerRateLimit` to the average user, but I am open to the change.

The scrape interval is user configurable. I imagine that it is rare that users change it. Many of the collector integration and stackrox e2e tests set it to a small value to make the tests faster.

Reply:
@JoukoVirtanen and I have discussed this a bit in our 1:1, and have come up with a possible solution:

If we move the `scrapeInterval` into the runtime config, then we can rename `perContainerRateLimit` to `maxConnsPerScrapeInterval` which might make the relationship a bit clearer due to their proximity. 

We could possibly also make the rate limit an absolute value rather than per-container, though I'd worry about the wider implications of that (e.g. one container with a lot of connections could suppress all network activity from other containers)

Reply:
`maxConnsPerScrapeInterval` (or even `maxConnsPerScrape` or `maxConnsPerInterval` SGTM). I think the "per container" part can be explained in the description.

Reply:
We are thinking of going with `maxConnectionsPerMinute` and then converting from connections per minute to connections per scrape in collector.

Reply:
cc @Stringy ^^